tensorwaves/data/generate.py

```diff
@@ -0,0 +1,109 @@
+"""Tools to facilitate data sample generation."""
+
+import logging
+
+import numpy as np
+
+from progress.bar import Bar
+
+from tensorwaves.interfaces import (
+    Function,
+    Kinematics,
+    PhaseSpaceGenerator,
+    UniformRealNumberGenerator,
+)
+
+
+def _generate_data_bunch(
```

Comment:
What precisely is a bunch? Even though this method is private, a docstring may solve this ambiguity (although perhaps I miss some jargon here).

Reply:
Hmm maybe chunk is a better word. Its only a small fix size sample. So depending on the requested total sample size, many of those bunches have to be created and concatenated. I can add a docstring though